src/engine.rs

```diff
@@ -138,21 +138,28 @@ where
 }
 
 // Adds a new stream to the engine.
-// TODO: handle possible errors here
```

Comment:
They aren't really handled well with Result so I wouldn't remove this TODO yet :)

Reply:
Aren't they all handled now consistently with rodio's fairly panic happy API? This does remove the unwrap that clearly could fail. Now we have panics that we generally wouldn't expect to happen, similar to elsewhere.